lib/async_experiments/experiment_result_control_worker.rb

```diff
@@ -0,0 +1,29 @@
+require "async_experiments/experiment_result"
+
+module AsyncExperiments
+  class ExperimentResultControlWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :experiments
+
+    LOCK_TIMEOUT = 60
+
+    def perform(name, id, run_output, duration, expiry, allowed_attempts = 5, attempt = 1)
```

Comment:
Oh I remember why I didn't now. You can't pass named arguments into a sidekiq worker, so instead we'd have to pass in a hash and then use `fetch` on it. Similarly it'd be worth keeping the `initialize` interface consistent with `AsyncExperiments::ExperimentResultCandidateWorker`. That still has 5 init arguments though. Feels like it'd add quite a bit of clutter though - so I think it might be better as an optimisation if the list of arguments grows further.